app/services/db/sp_cost/add_sp_cost.rb

```diff
@@ -13,7 +13,9 @@ class SpCostTypeError < StandardError; end
         acuant_result
         lexis_nexis_resolution
         lexis_nexis_address
+        mock_resolution
         gpo_letter
+        socure_resolution
```

Comment:
Checking with biz ops to make sure this new SP cost type doesn't make anything explode.

Reply:
✅ got the ok that sending a new sp cost type will not break anything